### ISSUE TITLE
QuickPulse Broken

### ISSUE THREAD

**Issue by debugthings** [Microsoft/ApplicationInsights-Java#360]
According to the code [here](https://github.com/Microsoft/ApplicationInsights-Java/blob/v1.0.7/core/src/main/java/com/microsoft/applicationinsights/internal/quickpulse/DefaultQuickPulsePingSender.java#L47-L68) you're explicitly building the JSON body using a string builder but it appears you're not surrounding the data with the curly braces. This breaks the expected format expected by the QuickPulse service.

**NOTE** that all of the errors come from the `com.microsoft.applicationinsights.core.dependencies.http.*` class. Which tells me that my QuickPulse telemetry is being counted as a dependency which should not happen as it's filtered out in other SDKs such as the .NET SDK [here](https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/v2.2.0/Src/DependencyCollector/Shared/Implementation/ApplicationInsightsUrlFilter.cs#L65)

Error From Stream
---------------------
```
Error converting value \"Instance\" to type 'Microsoft.ManagementServices.RealTimeDataProcessing.QuickPulseService.MonitoringDataPoint'. Path '', line 1, position 10.","Exception":""
```

Message Capture from Java Application
--------------------------------------------
```
"Instance":"192.168.1.145","InstrumentationKey":{{REMOVED}},"InvariantVersion":2,"MachineName":"192.168.1.145","Version":"2.2.0-424","StreamId":{{REMVOED}},"Documents":null,"Metrics":null,"Timestamp": "\/Date(1485820304291)\\/\""
```
------------------
Capture from .NET Application
---------------------------------
```
{
    "Documents": null,
    "Instance": "{{machinename}}.microsoft.com",
    "InstrumentationKey": null,
    "InvariantVersion": 2,
    "IsWebApp": false,
    "MachineName": "{{machinename}}",
    "Metrics": null,
    "StreamId": "{{REMOVED}}",
    "Timestamp": "\/Date(1485828129431)\/",
    "Version": "2.2.0-738"
}
```

Log File
---------
```
http-outgoing-0 >> POST /QuickPulseService.svc/ping?ikey=ping?ikey=6337ac5a-cc11-4f76-b9af-64b73f8ad671 HTTP/1.1
http-outgoing-0 >> x-ms-qps-transmission-time: 636214171042910000
http-outgoing-0 >> Content-Length: 275
http-outgoing-0 >> Host: rt.services.visualstudio.com
http-outgoing-0 >> Connection: Keep-Alive
http-outgoing-0 >> User-Agent: Apache-HttpClient/4.3.5 (java 1.5)
http-outgoing-0 >> Accept-Encoding: gzip,deflate
http-outgoing-0 >> "POST /QuickPulseService.svc/ping?ikey=ping?ikey={{REMOVED}} HTTP/1.1[\r][\n]"
http-outgoing-0 >> "x-ms-qps-transmission-time: 636214171042910000[\r][\n]"
http-outgoing-0 >> "Content-Length: 275[\r][\n]"
http-outgoing-0 >> "Host: rt.services.visualstudio.com[\r][\n]"
http-outgoing-0 >> "Connection: Keep-Alive[\r][\n]"
http-outgoing-0 >> "User-Agent: Apache-HttpClient/4.3.5 (java 1.5)[\r][\n]"
http-outgoing-0 >> "Accept-Encoding: gzip,deflate[\r][\n]"
http-outgoing-0 >> "[\r][\n]"
http-outgoing-0 >> ""Instance":"192.168.1.145","InstrumentationKey":{{REMOVED}},"InvariantVersion":2,"MachineName":"192.168.1.145","Version":"2.2.0-424","StreamId":{{REMVOED}},"Documents":null,"Metrics":null,"Timestamp": "\/Date(1485820304291)\\/\"""


http-outgoing-0 << "HTTP/1.1 500 Internal Server Error[\r][\n]"
http-outgoing-0 << "Content-Length: 322[\r][\n]"
http-outgoing-0 << "Content-Type: application/json; charset=utf-8[\r][\n]"
http-outgoing-0 << "Server: Microsoft-IIS/8.5[\r][\n]"
http-outgoing-0 << "x-ms-qps-subscribed: false[\r][\n]"
http-outgoing-0 << "Access-Control-Allow-Origin: *[\r][\n]"
http-outgoing-0 << "x-ms-request-target-ikey: {{REMOVED}}[\r][\n]"
http-outgoing-0 << "X-Powered-By: ASP.NET[\r][\n]"
http-outgoing-0 << "Strict-Transport-Security: max-age=31536000; includeSubDomains[\r][\n]"
http-outgoing-0 << "Date: Mon, 30 Jan 2017 23:51:44 GMT[\r][\n]"
http-outgoing-0 << "[\r][\n]"
http-outgoing-0 << "{"RequestId":"00000000-0000-0000-0000-000000000000","ResponseDateTime":"2017-01-30T23:51:44.5856395Z","Code":"InvalidOperation","Message":"Error converting value \"Instance\" to type 'Microsoft.ManagementServices.RealTimeDataProcessing.QuickPulseService.MonitoringDataPoint'. Path '', line 1, position 10.","Exception":""}"
```

**Comment by Hambik** [comment]
Same happened with me. any ETA here ?

**Comment by Dmitry-Matveev** [comment]
The current ETA is milestone 1.0.8. Unfortunately, it was not planned for the certain date yet.
You can keep an eye on milestone 1.0.8 progress (as soon as it gets a date, you can track the bug fixes within).

**Comment by Dmitry-Matveev** [comment]
Fixed in 1.0.8
